.github/workflows/CI-build-test-publish.yml

```diff
@@ -145,4 +145,43 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: "zkv_runtime.${{ inputs.chain }}.compact.compressed.wasm"
-          path: ./zkv_runtime.compact.compressed.wasm
\ No newline at end of file
+          path: ./zkv_runtime.compact.compressed.wasm
+
+      - name: Notify Slack Production Releases
+        uses: slackapi/slack-github-action@v1.25.0
+        with:
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": "🎉 zkVerify Docker Image v${{ github.ref_name }} Released! 🎉"
+                  }
+                },
+                {
+                  "type": "section",
+                  "fields": [
+                    {
+                      "type": "mrkdwn",
+                      "text": "*Docker Image:*  <https://hub.docker.com/r/horizenlabs/zkverify/tags?page=1&name=${{ github.ref_name }}| v${{ github.ref_name }}>\n"
+                    },
+                    {
+                      "type": "mrkdwn",
+                      "text": "*Build URL:*  <https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|View Build>"
+                    }
+                  ]
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "```docker pull horizenlabs/zkverify:${{ github.ref_name }}```"
```

Comment:
Please use env var `DOCKER_HUB_ORG` and `DOCKER_IMAGE_BUILD_NAME` if possible instead of hard-coded values. 

env vars are already available to all the whole pipeline

Reply:
Updated